target/linux/ramips/dts/mt7628an_trolink_w7628dna.dts

```diff
@@ -0,0 +1,52 @@
+#include "mt7628an.dtsi"
+
+/ {
+	compatible = "trolink,w7628dna", "mediatek,mt7628an-soc";
+	model = "Shenzen Trolink W7628DNA AP Module";
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x30000>;
+				read-only;
+			};
+
+			partition@30000 {
+				label = "u-boot-env";
+				reg = <0x30000 0x10000>;
```

Comment:
Please add the position of the U-Boot environment partition to `package/boot/uboot-envtools/files/ramips`. This will allow to use the `fw_printenv` tool.